roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -8,6 +8,12 @@
     "environment": {
         "KOLLA_CONFIG_STRATEGY":"COPY_ALWAYS"
     },
+{% if edpm_nova_healthcheck %}
+    "healthcheck": {
+        "test": "/openstack/healthcheck",
+        "mount": "/var/lib/containers/healthchecks/nova_compute"
+    },
+{% endif %}
```

Comment:
injecting executabels even if they are scripts is not supported as its a potential security issues so if we want to have a helthcheck script then it would be better to bake it into the container via tcib.


beyond that however i don't think we should do this today.

we could implement a health check via systems perhaps but we should avoid using the pdoman helthchec feature as that has caused oci runtime issues in the past.

from a product point of view we do not plan to provide container helthchecks until https://issues.redhat.com/projects/OSPRH/issues/OSPRH-7 is complete which is why its set as blocking https://issues.redhat.com/browse/OSPRH-5007

Reply:
Commented above. To add to it, I'm not sure if the lack of reporting health of one service should block all other services.

Reply:
given we effectively use the same pgrep query for now on the openshift side.

if gibi is ok with this ill retract my object to adding it for nova as long as the healthcheck action is None(this is the current default).
without https://issues.redhat.com/browse/OSPRH-6601 all other options would potentially break ongoing RPCs

Reply:
@gibizer